Dixit_Client/ViewModel/PictureConverter.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Data;
+
+using Dixit_Data;
+using System.Windows;
+using System.Windows.Media.Imaging;
+
+namespace Dixit_Client.ViewModel
+{
+    /// <summary>
+    /// Gets the picture's Id and returns the corresponding image
+    /// </summary>
+    class PictureConverter : IValueConverter
+    {
+        static CardAccess ca = new CardAccess();
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
```

Comment:
Okay, can we work with anything more specific than object? Like using Bitmap class or something